src/index.js

```diff
@@ -1,36 +1,116 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Wake up',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Pray',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Eat breakfast',
-    completed: false,
-    index: 2,
-  },
-];
-
+let tasks = [];
 const taskWrapper = document.querySelector('.to-dos');
+const newTask = document.querySelector('.new-task');
+const addNewTask = document.querySelector('.submit');
+const clearAll = document.querySelector('.clear-all');
 
-const showList = () => {
-  taskWrapper.innerHTML = '';
+const addToLocalStorage = () => {
+  localStorage.setItem('myTasks', JSON.stringify(tasks));
+};
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('myTasks')) {
+    tasks = JSON.parse(localStorage.getItem('myTasks'));
+  }
+  return tasks;
+};
+
+const resetIndex = (tasks) => {
   for (let i = 0; i < tasks.length; i += 1) {
-    taskWrapper.innerHTML += `
-    <form id="form">
-      <input type="checkbox" id="${tasks[i].index}" name="task" value="task">
-      <label for="${tasks[i].index}">${tasks[i].description}</label>
-      <i class="fa fa fa-times"></i><br>
-    </form>
-    `;
+    const indx = i + 1;
+    tasks[i].index = indx;
+  }
+};
+
+const rmvTask = (index) => {
```

Comment:
- [ ] When removing an item from the list, all remaining items' indexes should be updated, so they represent the current list order and are unique. For example, I removed item 2 but 3rd item's index still remained 3 instead of updating it to 2. You might notice that the feature is not implemented by looking at the below screenshot one more time 👇 
![Screenshot (199)_LI](https://user-images.githubusercontent.com/67087664/155325046-8c58b4aa-c32b-41bd-a711-2173d2f62fc9.jpg)

Kindly implement the necessary changes 🙏